src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const evacuateX = 1400;
+  const evacuateY = 500;
+
+  const robot = {
+    name,
+    version,
+    wheels,
+    'coords': {
```

Comment:
```suggestion
    coords: {
```
key of objects are string by default, there is no need to explicitly make them as strings